src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php

```diff
@@ -140,8 +140,14 @@ public function buildIterator()
                 'date' => $submission->getCreated()->format('d/m/Y H:i:s'),
                 'language' => $submission->getLang(),
             ];
+            $counter = 1;
             foreach ($submission->getFields() as $field) {
+                $fieldName = $field->getFieldName();
                 $fieldName = preg_replace('/\d+/', '', $field->getFieldName());
+                if(isset($data[$fieldName])) {
+                    $fieldName .= $counter;
```

Comment:
Can't you just use the `$field->getSequence()` method?

Reply:
Good point when i look at the database that should work. im gonna test it.